tf/env/staging/secrets-gcs.tf

```diff
@@ -22,3 +22,24 @@ resource "kubernetes_secret" "gcs-hmac-key" {
     "secret-key" = google_storage_hmac_key.dev-backup-upload-key.secret
   }
 }
+
+resource "google_storage_hmac_key" "public-assets-key" {
+  service_account_email = google_service_account.dev-api.email
+}
+
+resource "google_project_iam_member" "public-assets-upload" {
+  role    = "roles/storage.admin"
+  member  = "serviceAccount:${google_service_account.dev-api.email}"
+  project = local.project_id
```

Comment:
I might be totally misunderstanding the google docs here but I think this is probably much broader than we want.

As I understand it this would grant this staging key rights to do things like delete the production backup bucket. I'm looking at how broad it sounds from the [storage docs](https://cloud.google.com/storage/docs/access-control/iam-roles#:~:text=Storage%20Admin%20(roles/storage.admin))

I think this resource:  [google_storage_bucket_iam_member](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/storage_bucket_iam#google_storage_bucket_iam_member) is probably more likely what we want. This is analogous to how we grant rights to the state bucket (see [here](https://github.com/wmde/wbaas-deploy/blob/d84bf06e30368a59eacd4742297cc98bb3cba554/tf/env/production/terraform.tf#L19)) and is probably better than overwriting the whole policy like we do with the backup buckets (see [here](https://github.com/wmde/wbaas-deploy/blob/d84bf06e30368a59eacd4742297cc98bb3cba554/tf/modules/buckets/main.tf#L39)).

I imagine the place this should maybe live is in https://github.com/wmde/wbaas-deploy/blob/d84bf06e30368a59eacd4742297cc98bb3cba554/tf/modules/buckets/main.tf

Reply:
Fixed via https://github.com/wmde/wbaas-deploy/pull/1372